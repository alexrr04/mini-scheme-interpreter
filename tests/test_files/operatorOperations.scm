(define (main)
    (display (+ 3 5))
    (newline)
    (display (- 10 7))
    (newline)
    (display (* 6 7))
    (newline)
    (display (/ 20 4))
    (newline)
    (display (mod 10 3))
    (newline)
    (display (< 5 10))
    (newline)
    (display (< 10 5))
    (newline)
    (display (>= 10 10))
    (newline)
    (display (= 4 4))
    (newline)
    (display (<> 4 4))
    (newline)
)