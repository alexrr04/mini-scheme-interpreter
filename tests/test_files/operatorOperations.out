8
3
42
5
1
#t
#f
#t
#t
#f